### PR TITLE
GOVUKAPP-589 Global ON / OFF Switch Post Amigos Feedback

### PR DESCRIPTION
**Pull request by TimGJD**
- Fix font size and spacing
- Create reusable header view
- Update button text to new requirement
- Add new grey divider that has appeared on Analytics Consent screen design since Post Amigos
- Respect the notch
- Remove bottom padding from all views that are anchored to the bottom of the screen to match onboarding as requested

### CODE REVIEW

File: Production/govuk_ios/SwiftUIViews/HeaderView.swift [hunk 1]
```diff
@@ -0,0 +1,31 @@
+import SwiftUI
+
+struct HeaderView: View {
+    private let title: String
+    private let subheading: String
+
+    public init(title: String, subheading: String) {
```
Comment (batesyuk) [inline Production/govuk_ios/SwiftUIViews/HeaderView.swift:7]: No need for `public` here

File: Production/govuk_ios/SwiftUIViews/HeaderView.swift [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+        self.title = title
+        self.subheading = subheading
+    }
+
+    public var body: some View {
```
Comment (batesyuk) [inline Production/govuk_ios/SwiftUIViews/HeaderView.swift:12]: No need for `public` here?

**Review by batesyuk — CHANGES_REQUESTED** [review]

File: Production/govuk_ios/SwiftUIViews/HeaderView.swift (hunk 1 above)
Comment (TimGJD) [reply Production/govuk_ios/SwiftUIViews/HeaderView.swift]: Done

File: Production/govuk_ios/SwiftUIViews/HeaderView.swift (hunk 2 above)
Comment (TimGJD) [reply Production/govuk_ios/SwiftUIViews/HeaderView.swift]: Done

**Review by batesyuk — APPROVED** [review]

**Pull request merged**